README.md

```diff
@@ -58,3 +58,22 @@ command:
 ```
 firefox com.ibm.streamsx.metrics/doc/spldoc/html/index.html &
 ```
+
+### Building Project
+The MetricsSink operator requires you to build jars and include them in your project. You will also need to have an ElasticSearch server running.
```

Comment:
I followed the example of the [streamsx.mail](https://github.com/IBMStreams/streamsx.mail) toolkit. It might be to make the toolkit more compact. I can include the jars in my next push.